device/esp_tinyusb/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
-## 1.4.4 (Unreleased)
+## 1.4.4
 
+- esp_tinyusb: Add BValid signal test
```

Comment:
* we usually only mention changes relevant for users. So we no release notes for tests